### ISSUE TITLE
Decide what to put in here

### ISSUE THREAD

**Issue by guFalcon** [frux-technologies/lobash#1]

**Comment by ghost** [comment]
The lobash library isn't published yet ;)

**Comment by guFalcon** [comment]
Yup. I know... We're still in the process of discussing what to put into our public code-repos... This lib exists, but it's not been decided to put it up yet. Sorry. We're working on it :)
